playground/webpack.config.js

```diff
@@ -39,4 +38,4 @@ module.exports = {
     filename: 'bundle.js',
     path: path.resolve(__dirname, 'dist'),
   },
-};
\ No newline at end of file
+};
```

Comment:
we should probably add `pre-commit` here too, lets see if that gets merged on `miden-vm` and then add here too.

Reply:
Got it, can we do that in a separate PR? This should just fix the build.